src/boxes/mp4_box.rs

```diff
@@ -55,16 +63,39 @@ impl std::fmt::Debug for dyn Mp4Box {
                 write!(f, "{:?}", internal)?;
             }
         }
+
+        let off_print = self.offsets();
```

Comment:
Having the offset for all the modules added this extra code here. Maybe you can handle this in the `stco` module?

Reply:
This has been resolved in the latest commit.